src/Python/sirf/contrib/BSREM/BSREM.py

```diff
@@ -14,7 +14,16 @@
 from cil.optimisation.algorithms import Algorithm 
 
 class BSREMSkeleton(Algorithm):
-    def __init__(self, data, initial, initial_step_size, relaxation_eta, **kwargs):
+    ''' Main implementation of a modified BSREM algorithm
+
+    This essentially implements constrained preconditioned gradient ascent
+    with an EM-type preconditioner
+    '''
+    def __init__(self, data, initial, initial_step_size, relaxation_eta,
+                 iteration_filter=STIR.TruncateToCylinderProcessor(), **kwargs):
+        '''
+        iteration_filter is applied after every update. Set it to `None` if you don't want any.
```

Comment:
I'm not sure I understand "after every update". 
```suggestion
        In each update step, the gradient of a subset is taken, before this multiplied by a step_size and a EM-type preconditioner. Before adding this to the previous iterate, an iteration_filter can be applied to or set to `None` if you don't want any.
```

Reply:
addressed